lua/boo/init.lua

```diff
@@ -32,20 +37,35 @@ function M.boo()
 		})
 	end
 	if config.close_on_mouse_move then
-		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI' }, {
+		vim.api.nvim_create_autocmd({ 'CursorMoved', 'CursorMovedI', 'InsertEnter' }, {
 			buffer = 0,
 			desc = 'Closes boo when moving the cursor',
 			group = vim.api.nvim_create_augroup('Closeboo', { clear = true }),
 			callback = close_callback,
 		})
 	end
+	if config.open_on_mouse_hold then
+		vim.api.nvim_create_autocmd({ 'CursorHold', 'InsertLeave' }, {
+			buffer = 0,
+			desc = 'Open boo when holding cursor',
+			group = vim.api.nvim_create_augroup('Openboo', { clear = true }),
+			callback = open_callback,
+		})
+	end
+
+	-- vim.api.nvim_create_autocmd({ 'WinLeave', 'BufWinLeave', 'BufDelete', 'ModeChanged', 'InsertEnter' }, {
+	-- 	buffer = 0,
+	-- 	desc = 'Closes boo on various conditions',
+	-- 	group = vim.api.nvim_create_augroup('ClosebooOnEvents', { clear = true }),
+	-- 	callback = close_callback,
+	-- })
```

Comment:
Before merging, could you remove this comment please :)

Reply:
i was going to tell u to debug that because it stays open like if im in a buffer and i do `]b` i go to the next buffer but it doesnt close so i was doing this autocmd which didnt work out well